src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  count: 0,
+};
+
+export const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      state.count += 1;
+    },
+    removeBook: (state) => {
+      state.count -= 1;
+    },
+  },
```

Comment:
- [ ] The parameters should also have action as the second parameter then use filter is a manner similar to `state.books = state.books.filter((book) => book.id !== action.id)`